CMakeLists.txt

```diff
@@ -53,7 +53,7 @@ option(FORCE_WINDOWS "Instructs the build system to force Windows builds when WI
 option(DISABLE_CURL "Disables libCurl Properties." OFF)
 
 option(USE_GOLD_LINKER "Use Gold Linker" OFF)
-
+option(INSTALLER_MERGE_MODULES "Creates installer with merge modules" OFF)
```

Comment:
Currently we build with merge modules. This changes to shipped dlls by default. Do we want that?

If you change the PR to the old behavior, please have a look at win_build_vs.bat as well.

Reply:
I am fine with it, I feel that the general direction is we want to go forward with the DLL-bundle version once we are sure that it works everywhere/deprecate support everywhere where it doesn't, but for now we have the ability to use the merge module version with a simple configuration.